README.rst

```diff
@@ -69,3 +71,21 @@ Quick start
         ...
         'accountsplus.middleware.TimezoneMiddleware',
     )
+
+7. As a security enhancement, you could enable blocking users from login after a specified number of failing login trials::
+
+    - To enable locking out through setting the following in your base.py::
+
+        RECAPTCHA_PUBLIC_KEY = <Key generated from Google Recaptcha API>
+        RECAPTCHA_PRIVATE_KEY = <Key generated from Google Recaptcha API>
+        RECAPTCHA_USE_SSL = True
+
+        AXES_LOCK_OUT_AT_FAILURE = True
+        AXES_COOLOFF_TIME = <Integer number of hours to block user>
+        AXES_LOGIN_FAILURE_LIMIT = <Count of failing attempts before lockout>
+        AXES_LOCKOUT_URL = 'locked/'
+        AXES_LOCKOUT_TEMPLATE = <Template you want to show on lockout>
```

Comment:
@nasief Please add references to `django-axes` and `django-recaptcha` libraries' documentations.